src/test/java/com/synopsys/integration/detect/battery/tests/NugetBattery.java

```diff
@@ -0,0 +1,23 @@
+package com.synopsys.integration.detect.battery.tests;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import com.synopsys.integration.detect.battery.BatteryTest;
+import com.synopsys.integration.detect.configuration.DetectProperty;
+
+@Tag("battery")
+public class NugetBattery {
+    @Test
+    void dotnetcore() {
+        final BatteryTest test = new BatteryTest("nuget-dotnetcore");
+        test.sourceDirectoryNamed("windows-nuget4");
+        test.sourceFileNamed("example.sln");
+        test.executableThatCopiesFiles(DetectProperty.DETECT_DOTNET_PATH, "NUGET-0")
+            .onWindows(5, "")
+            .onLinux(3, "--output_directory=");
```

Comment:
What are the 5 and 3? If I am making the test an I am on a Mac, how will I know what to put for Windows and vise versa?

Reply:
This is the least developer friendly part of these tests - there isn't really a way to know intuitively. Windows splits on "=" so it is off by the number of equal signs and doesn't start with the same prefix. I don't have a better way to do it.